### ISSUE TITLE
Recent additions to the meta server are not showing up on metaserver.us.cross-fire.org

### ISSUE THREAD

**Issue by basictheprogram** [basictheprogram/crossfire-metaserver#2]
Several new crossfire servers are not show up on metaserver.us.cross-fire.org

**Comment by basictheprogram** [comment]
cc: @tannerrj

**Comment by basictheprogram** [comment]
Several column have size restrictions.
```sql
mysql> describe servers ;
+--------------+---------------+------+-----+---------+----------------+
| Field        | Type          | Null | Key | Default | Extra          |
+--------------+---------------+------+-----+---------+----------------+
| entry        | int(11)       | NO   | PRI | NULL    | auto_increment |
| hostname     | varchar(80)   | YES  |     | NULL    |                |
| port         | int(11)       | YES  |     | NULL    |                |
| html_comment | varchar(1024) | YES  |     | NULL    |                |
| text_comment | varchar(256)  | YES  |     | NULL    |                |
| archbase     | varchar(20)   | YES  |     | NULL    |                |
| mapbase      | varchar(20)   | YES  |     | NULL    |                |
| codebase     | varchar(20)   | YES  |     | NULL    |                |
| flags        | varchar(20)   | YES  |     | NULL    |                |
| num_players  | int(11)       | YES  |     | NULL    |                |
| in_bytes     | int(11)       | YES  |     | NULL    |                |
| out_bytes    | int(11)       | YES  |     | NULL    |                |
| uptime       | int(11)       | YES  |     | NULL    |                |
| version      | varchar(20)   | YES  |     | NULL    |                |
| sc_version   | varchar(20)   | YES  |     | NULL    |                |
| cs_version   | varchar(20)   | YES  |     | NULL    |                |
| last_update  | datetime      | YES  |     | NULL    |                |
+--------------+---------------+------+-----+---------+----------------+
```
Several servers exceed those sizes

archbase
```
echo "Weather branch+24x24 face set" | wc -c
30
```
version
```
echo “v1.75.0-1422-gbd88ab894” | wc -c
24
```

**Comment by basictheprogram** [comment]
New schema

```sql
+--------------+---------------+------+-----+---------+----------------+
| Field        | Type          | Null | Key | Default | Extra          |
+--------------+---------------+------+-----+---------+----------------+
| entry        | int(11)       | NO   | PRI | NULL    | auto_increment |
| hostname     | varchar(80)   | YES  |     | NULL    |                |
| port         | int(11)       | YES  |     | NULL    |                |
| html_comment | varchar(1024) | YES  |     | NULL    |                |
| text_comment | varchar(256)  | YES  |     | NULL    |                |
| archbase     | varchar(64)   | YES  |     | NULL    |                |
| mapbase      | varchar(20)   | YES  |     | NULL    |                |
| codebase     | varchar(64)   | YES  |     | NULL    |                |
| flags        | varchar(20)   | YES  |     | NULL    |                |
| num_players  | int(11)       | YES  |     | NULL    |                |
| in_bytes     | int(11)       | YES  |     | NULL    |                |
| out_bytes    | int(11)       | YES  |     | NULL    |                |
| uptime       | int(11)       | YES  |     | NULL    |                |
| version      | varchar(64)   | YES  |     | NULL    |                |
| sc_version   | varchar(20)   | YES  |     | NULL    |                |
| cs_version   | varchar(20)   | YES  |     | NULL    |                |
| last_update  | datetime      | YES  |     | NULL    |                |
+--------------+---------------+------+-----+---------+----------------+
```

**Comment by basictheprogram** [comment]
Dup of https://sourceforge.net/p/crossfire/bugs/960/

Fixed with 60d61f45710165a97042da16276386645585b608
